relay: Next iteration of refactoring (#2365)
getsentry/sentry-docs: src/docs/product/relay/monitoring/index.mdx

```
INFO  relay::setup > launching relay from config folder .relay
INFO  relay::setup >   relay mode: managed
INFO  relay::setup >   relay id: cde0d72e-0c4e-4550-a934-c1867d8a177c
INFO  relay::setup >   log level: INFO
```

This example displays the messages with the default logging level (`INFO`), which you can modify so it displays either more or less information. For details about configuring logging please see the [Logging](/product/relay/options/#logging) section on the options page.

## Error Reporting

By default, Relay logs errors to the configured logger. You can enable error
reporting to your project in Sentry in the Relay configuration file:

```yaml
sentry:
  enabled: true
  dsn: <your_dsn>
```

More information about available options and their meaning can be found [on the options page](/product/relay/options/#internal-error-reporting).

## Health Checks

Relay provides two URLs for checking system status:

- `GET /api/relay/healthcheck/live/`: Tests if Relay is running and listening to
  HTTP requests.
- `GET /api/relay/healthcheck/ready/`: Tests if Relay is authenticated with the
  upstream and operating normally.

> In case of success, both endpoints return a _200 OK_ response:

Reviewer: ```suggestion
Both endpoints return a _200 OK_ response when successful:  
```

```json
{
  "is_healthy": true
}
```

## Metrics

> Stats can be submitted to a StatsD server by configuring the `metrics.statsd` key. The key

Reviewer: ```suggestion
You can submit stats to a StatsD server by configuring the `metrics.statsd` key to an `ip:port` tuple.
```

can be set to an `ip:port` tuple.

### Example Configuration

```yaml
metrics:
  # Endpoint of your StatsD server
  statsd: 127.0.0.1:8126
  # Prefix all metric names with this string
  prefix: mycompany.relay
```

> All available options for configuring metrics reporting can be found on the [options page](/product/relay/options/#statsd-metrics).

Reviewer: ```suggestion
The options for configuring metrics reporting are documented on the [options page](/product/relay/options/#statsd-metrics).
```
